Tools/autotest/arducopter.py

```diff
@@ -5148,6 +5148,31 @@ def test_parameter_validation(self):
         self.drain_mav()
         self.assert_prearm_failure("Check MOT_PWM_MIN/MAX")
 
+    def test_alt_estimate_prearm(self):
+        self.context_push()
+        ex = None
+        try:
+            self.set_parameter("EK2_ALT_SOURCE", 2)
+            self.wait_gps_disable()
```

Comment:
We should change this to not be of the "wait_" signature, which don't typically make changes, just wait for something.

Not this PR, 'though.

Reply:
agree! we should a set and a wait function